packages/components/src/KaizenProvider/_docs/KaizenProvider.mdx

```diff
@@ -6,7 +6,7 @@ import migrationFETToV1 from "!!raw-loader!./code-diffs/migration-fe-template-to
 import migrationFETBonus from "!!raw-loader!./code-diffs/migration-fe-template-bonus.diff"
 import { KaioNotification } from "../../../../../storybook/components"
 
-<Meta title="Components/🛠️ Kaizen Provider/Installation" />
+<Meta title="Components/Kaizen Provider/Installation" />
 
 # Kaizen Provider
```

Comment:
Similar to how we link to the page for "Managing the theme", we need one for Internationalisation